src/user/user.service.ts

```diff
@@ -60,6 +63,21 @@ export class UserService {
         return user;
     }
 
+    public async requestToBeTester(
+        testFlightRequest: TestFlightRequest
+    ): Promise<Tester> {
+        const requestTest = await this.testerRepository.findOneBy({
+            email: testFlightRequest.email.toLowerCase()
+        });
+        if (!requestTest) {
```

Comment:
can be made cleaner using guard clause & early return

if(requestTest) return requestTest

... rest